loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -243,7 +248,8 @@ static int binary_update_download_new_binary(void)
 	ret = write(write_fd, (FAR uint8_t *)&header_data, sizeof(binary_header_t));
 	if (ret != sizeof(binary_header_t)) {
```

Comment:
If it tries to write some data, but not sizeof(binary_header_t), some data will be written, but not erased.
Is it okay?